components/greaselion/browser/BUILD.gn

```diff
@@ -11,6 +11,10 @@ source_set("browser") {
     "switches.h",
   ]
 
+  configs += [
+    "//brave/components/brave_component_updater/browser:network_config"
```

Comment:
I would just make this a public config of `brave/components/brave_component_updater/browser`